employer-engagement/scoring/non_levy_model_score.py

```diff
@@ -0,0 +1,253 @@
+
+# Function for new company flag
+
+def fn_new_company(row):
+    if row['months_since_sign_up2']<=6 :
+        val=1
+    else:
+        val=0
+    return val
+
+non_levy_score_set['new_company']=non_levy_score_set.apply(fn_new_company,axis=1)
+
+# Only keep relevant variables and rename accordingly
+model_cols_to_keep=['A1','A3','months_since_sign_up2','occupation_1', \
+                    'occupation_2','occupation_3','occupation_7','occupation_13','occupation_14','occupation_15', \
+                    'occupation_17','occupation_20','occupation_22','occupation_24','occupation_null','employees', \
+                    'years_since_tpr_signup','comp_type_C','comp_type_E','comp_type_F','comp_type_I','comp_type_L', \
+                    'comp_type_P','comp_type_S','comp_type_X','tpr_match','new_company','early_adopter', \
+                    'commitments_ending_12m','prev_12m_new_commitments','prev_12m_new_levy_transfers', \
+                    'levy_sending_company','current_live_commitments','company_status']
+non_levy_score_set = non_levy_score_set[model_cols_to_keep]
+non_levy_score_set.columns = ['levy_non_levy','account_id','as_months_since_sign_up','occupation_1', \
+                     'occupation_2','occupation_3','occupation_7','occupation_13','occupation_14','occupation_15', \
+                     'occupation_17','occupation_20','occupation_22','occupation_24','occupation_null','employees', \
+                     'years_since_tpr_signup','comp_type_C','comp_type_E','comp_type_F','comp_type_I','comp_type_L', \
+                     'comp_type_P','comp_type_S','comp_type_X','tpr_match','new_company','early_adopter', \
+                     'commitments_ending_12m','prev_12m_new_commitments','prev_12m_new_levy_transfers', \
+                     'levy_sending_company','current_live_commitments','company_status']
+
+# Take logs to standardise the scale
+non_levy_score_set['log_adjusted_commitments'] = np.log2(non_levy_score_set['adjusted_commitments']+1)
+non_levy_score_set['log_employees'] = np.log2(non_levy_score_set['employees']+1)
+
+
+print(non_levy_score_set)
+
+
+# Select model variables only
+
+# Take logs to standardise the scale
+non_levy_score_set['log_employees'] = np.log2(non_levy_score_set['employees']+1)
+
+X = non_levy_score_set[['levy_non_levy','as_months_since_sign_up','occupation_1', \
+                     'occupation_2','occupation_3','occupation_7','occupation_13','occupation_14','occupation_15', \
+                     'occupation_17','occupation_20','occupation_22','occupation_24','occupation_null','employees', \
+                     'years_since_tpr_signup','comp_type_C','comp_type_I', \
+                     'comp_type_X','tpr_match','new_company','early_adopter', \
+                     'commitments_ending_12m','prev_12m_new_commitments','prev_12m_new_levy_transfers', \
+                     'levy_sending_company','current_live_commitments','company_status']]
+
+
```

Comment:
Logging appears to be inconsistent across model_score.py files, e.g. comparing with

https://github.com/SkillsFundingAgency/das-data-management-ai/pull/3/files#diff-f9a39c5e77ef3a3abbe02955cb1b1fd072f0e00e61b168ca080f4c8ac31d3413R263-R282